core/src/trezor/ui/components/tt/info.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         style: InfoConfirmStyleType = DEFAULT_STYLE,
     ) -> None:
         super().__init__()
-        self.text = text.split()
+        self.text = text
```

Comment:
```suggestion
        self.text = [text]
```

Reply:
very subtle but it's a bug! `render_text` expects an iterable of strings. if we pass the original text, the iterable is `["h", "e", "l"....]`, instead of `["hello"]` which we want

with this fixed and bounds tweaked, the automatic linebreaker does a decent job

![image](https://user-images.githubusercontent.com/596073/120649739-6175d380-c47d-11eb-8c45-fa279b214230.png)

Reply:
oof, good catch, i feel retarded ... should be fixed in 38a48e2b374c0ff8461160f76c3c31cbad9f7741